wid/gatt.py

```diff
@@ -1690,7 +1690,7 @@ def hdl_wid_141(params: WIDParams):
 
     hdl1 = MMI.args[0]
     hdl2 = MMI.args[1]
-    btp.gattc_read_multiple(btp.pts_addr_type_get(), btp.pts_addr_get(), hdl1, hdl2)
```

Comment:
Might be worth seeing if this is not the case where PTS uses the same WID for two different actions. 
Can you check GATT/CL/GAR/BV-05-C and GATT/CL/GAR/BI-[18-22]-C with this change?

Reply:
They seem to work:
```
GATT    GATT/CL/GAR/BI-18-C             PASS
GATT    GATT/CL/GAR/BI-19-C             PASS
GATT    GATT/CL/GAR/BI-20-C             PASS
GATT    GATT/CL/GAR/BI-21-C             PASS
GATT    GATT/CL/GAR/BI-22-C             PASS
GATT    GATT/CL/GAR/BV-05-C             PASS
```

Reply:
Thanks!